CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Cromwell Change Log
 
+## 45 Release Notes
+
+### List dependencies flag in WOMtool Command Line 
+
+WOMtool now outputs the list of files referenced in import statements using `-l` flag for `validate` command.
```

Comment:
When we (really I) have written about Womtool in the changelog, I didn't capitalize "WOM"... but I see our docs refer to "WOMtool"

Potential tiebreaker in favor of "Womtool": users generally don't know what "WOM" is an acronym for, so they probably don't understand why we capitalized it

Reply:
Hmm it does make sense to change it to `Wom`tool if users don't know what `WOM` stands for. Will update it 👍